src/backend/storeManagers/gog/presence.ts

```diff
@@ -0,0 +1,83 @@
+import { app } from 'electron'
+import { logError, logInfo, LogPrefix } from 'backend/logger/logger'
+import { axiosClient } from 'backend/utils'
+import { GOGUser } from './user'
+import { isOnline } from 'backend/online_monitor'
+
+interface PresencePayload {
+  application_type: string
+  force_update: boolean
+  presence: 'online' | 'offline'
+  version: string
+  game_id?: string
+}
+
+let CURRENT_GAME = ''
+let interval: NodeJS.Timeout
+
+function setCurrentGame(game: string) {
+  CURRENT_GAME = game
+}
+
+async function setPresence() {
+  try {
+    if (!GOGUser.isLoggedIn() || !isOnline) return
+    const credentials = await GOGUser.getCredentials()
+    if (!credentials) {
+      return
+    }
```

Comment:
```suggestion
    if (!credentials) return
```

to match the syntax in line 24 doing a single line if/return